docs/HowTo/installation/configure-klaw-wizard.md

```diff
@@ -56,7 +56,7 @@ URL: [https://host:port/helpwizard]
    Klaw is multi-tenant by default. Every tenant has its own set of Kafka
    environments, and users of one tenant cannot view/access topics, or ACLs
    from other tenants. To configure tenants, navigate to **Dashboard -\>
-   Settings**, and look for [klaw.tenant.config].
+   Settings**, and look for [Klaw.tenant.config].
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one?